### PR TITLE
Be able to add some annotations

### PR DESCRIPTION
**Pull request by sbrunner**
That can be used by Operators

### CODE REVIEW

File: tests/qgis-expected.yaml [hunk 1]
```diff
@@ -69,6 +69,8 @@ metadata:
     app.kubernetes.io/name: apache
     app.kubernetes.io/instance: qgis
     app.kubernetes.io/component: main
+  annotations:
+    reloader.stakater.com/auto: "true"
```
Comment (danduk82) [inline tests/qgis-expected.yaml:73]: for info: what is this?

**Review by danduk82 — APPROVED** [review]
ok

File: tests/qgis-expected.yaml (hunk 1 above)
Comment (sbrunner) [reply tests/qgis-expected.yaml:73]: As example, I use the annotation used by https://github.com/stakater/Reloader#how-to-use-reloader

**Comment by sbrunner** [comment]
Thanks :-)

**Pull request merged**